src/shared/codelens/codeLensUtils.ts

```diff
@@ -22,7 +22,7 @@ export type Language = 'python' | 'javascript'
 
 export interface CodeLensProviderParams {
     configuration: SettingsConfiguration,
-    toolkitOutputChannel: vscode.OutputChannel, // TODO: Rename this lambdaOuputChannel? ouputChannel? Provide both?
+    toolkitOutputChannel: vscode.OutputChannel, // TODO: Rename this lambdaOutputChannel? outputChannel? Provide both?
```

Comment:
🙂

Reply:
Interpreted as rename to `outputChannel` externally we can go back and forth any number of time making more or less output channels, this function need not know about that.
✔️

Reply:
I was just happy because spelling makes me happy :) No action required.